packages/esm-active-visits-app/src/active-visits-widget/active-visits.test.tsx

```diff
@@ -52,7 +53,9 @@ describe('ActiveVisitsTable', () => {
     expect(patientNameLink.tagName).toBe('A');
   });
 
-  it.skip('filters active visits based on search input', () => {
+  it.skip('filters active visits based on search input', async () => {
```

Comment:
Should we still be skipping this test or should we just get rid of it?

Reply:
Neither. Fixed in the latest commit.

Reply:
Ah! Much better!